### PR TITLE
Fix broken links in `README.md`

### PR DESCRIPTION
**Pull request by hvy**
## Motivation

See description.

## Description of the changes

Fix broken links in `README.md`.

### CODE REVIEW

**Comment by hvy** [comment]
@toshihikoyanase, @HideakiImamura could you review this PR? I think changes are pretty straight forward.

File: README.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ Optuna can dynamically construct the search spaces for the hyperparameters.
...
+- [Pythonic search spaces](https://optuna.readthedocs.io/en/stable/tutorial/10_key_features/002_configurations.html)
   - Define search spaces using familiar Python syntax including conditionals and loops.
-- [Efficient optimization algorithms](https://optuna.readthedocs.io/en/stable/tutorial/007_pruning.html)
+- [Efficient optimization algorithms](https://optuna.readthedocs.io/en/stable/tutorial/10_key_features/003_efficient_optimization_algorithms.html)
   - Adopt state-of-the-art algorithms for sampling hyper parameters and efficiently pruning unpromising trials.
-- [Easy parallelization](https://optuna.readthedocs.io/en/stable/tutorial/004_distributed.html)
+- [Easy parallelization](https://optuna.readthedocs.io/en/stable/tutorial/10_key_features/004_distributed.html)
   - Scale studies to tens or hundreds or workers with little or no changes to the code.
 - [Quick visualization](https://optuna.readthedocs.io/en/stable/reference/visualization/index.html)
   - Inspect optimization histories from a variety of plotting functions.
```
Comment (toshihikoyanase) [inline README.md:43]: We have a tutorial page for the visualization feature now, and I think we can replace this like with it.
Should we work on it in a follow-up PR?

https://optuna.readthedocs.io/en/stable/tutorial/10_key_features/005_visualization.html

**Review by toshihikoyanase — COMMENTED** [review]
Thank you for your PR. The change looks good to me.
I added a suggestion on "Quick visualization."

File: README.md (hunk 1 above)
Comment (hvy) [reply README.md]: You're right. It's also already updated at the [key features in the docs](https://optuna.readthedocs.io/en/stable/). I just fixed the URL.

**Comment by hvy** [comment]
PTAL.

**Review by HideakiImamura — APPROVED** [review]
LGTM.

**Review by toshihikoyanase — APPROVED** [review]
LGTM. Thank you for your update.

**Pull request merged**